scripts/kustomize.sh

```diff
@@ -0,0 +1,86 @@
+#!/bin/bash
```

Comment:
Maybe change this file to `install_kustomize.sh`

Reply:
As stated above: this is not a "installer". It is supposed to be called instead of the kustomize binary. It will cache the real binary and "exec" it.